### DISCUSSION TITLE
Docker Compose是否有参数可以设定默认的System Assistants

### DISCUSSION THREAD

**Discussion by SneakyTurt1e** [lobehub/lobehub#4758] [Self Hosting] [answered]
我在文档中找到了默认助手的变量并成功设置，但是我没有找到设置System Assistants的方法，我并没有使用GPT。

![image](https://github.com/user-attachments/assets/ef7303d7-7fbe-43b6-a481-c012eb00c154)

**Comment by SpeedupMaster** [comment, marked as answer]
#4735
`SYSTEM_AGENT="agentMeta=zhipu/glm-4-flash,historyCompress=zhipu/glm-4-flash,queryRewrite=zhipu/glm-4-flash,topic=zhipu/glm-4-flash,translation=zhipu/glm-4-flash"`
